docs/src/idiomatic/README.md

```diff
@@ -51,7 +51,7 @@ reduce your discomfort when learning more advanced TLA+.
 __Idiom 0:__
     [Keep state variables to the minimum](000keep-minimum-state-variables.md) :battery:
 
-__Idiom 1:__ [Update state variables with assignments](001assignments.md) :date:
+__Idiom 1:__ [Update state variables with assignments](assignments.md) :date:
```

Comment:
Should this be in the apalache user manual rather than in the idioms guidelines? It seems more suitable to the former.

Reply:
Yeah, Igor and I already discussed this, it's probably getting moved to somewhere in the main body. I'm waiting on his review to figure out where exactly.

Reply:
Seems like an `assignments.md` chapter under `apalache` would make the most sense to me.